.swiftformat

```diff
@@ -1,4 +1,4 @@
---exclude **/.build,**/Carthage,**/DerivedData,**/AutoMockable.generated.swift,WireUI/Sources/WireDesign/Icons/Autogenerated/StyleKitIcons.generated.swift
+--exclude **/.build,**/Carthage,**/DerivedData,**/AutoMockable.generated.swift,WireUI/Sources/WireDesign/Icons/Autogenerated/StyleKitIcons.generated.swift,wire-ios-automation
```

Comment:
I think this should be `wire-ios-automation/**`.

However please first merge `develop` into your branch as `.swiftformat` has changed quite a lot

Reply:
Yeah you are right. And the new format is far more readable